### PR TITLE
Add flanders ip util v1.8.2

### PR DESCRIPTION
**Pull request by missionology**
After making all changes to the cask:

- [Y] `brew cask audit --download {{cask_file}}` is error-free.
- [Y] `brew cask style --fix {{cask_file}}` reports no offenses.
- [Y] The commit message includes the cask’s name and version.
- [Y] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [Y] Named the cask according to the [token reference].
- [Y] `brew cask install {{cask_file}}` worked successfully.
- [Y] `brew cask uninstall {{cask_file}}` worked successfully.
- [Y] Checked there are no [open pull requests] for the same cask.
- [Y] Checked the cask was not [already refused].
- [Y] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/flanders-ip-utility.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'flanders-ip-utility' do
```
Comment (vitorgalvao) [inline Casks/flanders-ip-utility.rb:1]: Does not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

File: Casks/flanders-ip-utility.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  version '1.8.2'
+  sha256 '2c6bdbda882d52484891460f0e84cd5c3b77fa8973e6d15cb6b9c4066bc7b465'
+
+  url "https://www.flandersscientific.com/ip-remote/release/IPRemoteUtility-#{version}-macOSX.zip"
```
Comment (vitorgalvao) [inline Casks/flanders-ip-utility.rb:5]: Use `https://flandersscientific.com/lut/mac.php` as an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Tip: use `x` in checklists (no spaces inside the brackets) and [Github will make them clickable](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments).

**Comment by missionology** [comment]
Hi thanks for the tip and help, I am new to contributing to open source.

I fixed the token reference, but I did not update the app-cast as the link you posted is for an unrelated app so if it needs an appcast I could add the same homepage link again? https://flandersscientific.com/ip-remote/

Thank you

**Comment by suschizu** [comment]
@missionology  - this url doesn't seem to work - do you perhaps have another project-url? 
`curl: (7) Failed to connect to flandersscientific.com port 443: Operation timed out`

**Comment by missionology** [comment]
Hi this works for me still,

Could it be geographical ? Where are you pulling from? Can you download it
directly from the site?

Https://flandersscientific.com/ip-remote

**Comment by suschizu** [comment]
hm that's strange - the website just works via VPN for me

**Pull request merged**